### ISSUE TITLE
508-defect-2: Content is not logically organized (including headings)

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#47355]
## Point of contact

Josh Kim

## Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

## Details

Headings are inconsistently applied throughout the page. Specifically...

**1. The following are not headings despite starting separate sub-sections of "prepare"**
  - `When you apply, be sure to have these on hand` 
  - `What if I need help filling out my application?`
  
**2. The second "sign in to start your application" button falls hierarchically under step 4 "Decision"**
  - This should be nested under step 2, not step 4
  
**3. Respondent burden also falls hierarchically under step 4 "Decision" as it doesn't have its own heading**
  - This can be resolved by providing it a unique heading
  - Respondent burden also appears in each of the form's pages (you can view this by starting the form flow without signing in)

## Reproduction steps

1. [Go to the apply for pre-need page](https://www.va.gov/burials-and-memorials/pre-need/form-10007-apply-for-eligibility/introduction)
2. Using [headingsmap](https://chrome.google.com/webstore/detail/headingsmap/flbjommegcjonpdmenkdiocclhjacmbi?hl=en) or a screen reader, confirm that headings are missing from the page

## Proposed solution or next steps

- [x] [Submit a review ticket to the content team](https://github.com/department-of-veterans-affairs/va.gov-team/issues/47358) to update the language on this page, including the heading structures
- [ ] Make updates to the page based on content team feedback
- [ ] A11y specialist to confirm changes have been made to close this ticket 

## References, articles, or WCAG support

1. [Accessible heading structures (a11y project) ](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/)

## Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [X] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by joshkimux** [comment]
@MichelleDieudonne heading issues on this page remain and are blocked until Randi's team is able to begin rework in the future. 

Tagging @RLHecht and @coforma-terry for awareness ❤️ + a quick question:
- What would be the right next step here? When I was still supporting this team, I marked this as blocked, but kept the product labels so it wouldn't get lost.

**Comment by ccruzgra** [comment]
@joshkimux I was curious if a similar content team rewriting was planned for the index/start page for the pre-need application: https://www.va.gov/burials-memorials/pre-need-eligibility/ since there may be similar issues with the headings.

**Comment by joshkimux** [comment]
It's the same folk! They have a content roadmap to prioritize what content to re-write.

**Comment by sonntag-adam** [comment]
@coforma-terry @SarahKay8 fix is in staging.

**Comment by sonntag-adam** [comment]
@SarahKay8 have you gotten a chance to see if this is good on your end?

**Comment by SarahKay8** [comment]
![C5D50B30-1241-4F1E-B25A-B71602B45CAA.jpeg](https://images.zenhubusercontent.com/623cd4d6774110eeb19922cb/bd37502d-c9b6-4a13-88d5-e967e0727c0e)

Confirmed that heading structure is consistent. Closing @sonntag-adam
